content/community/governance/gcb/index.md

```diff
@@ -1,114 +1,63 @@
-  * Details of access to the SIG’s communication channels.
-  * A description of roadmap-related processes for gathering input from members.
-  * The current contact/email address for the nominated GCB representative, and to manage membership and internal communication for itself. 
-* Use at least one public Matrix channel under the Galaxy Matrix space.
+Galaxy Governance consists of a [Galaxy Executive Board](https://galaxyproject.org/community/governance/geb/) that provides global direction, working with a [Galaxy Technical Board](https://galaxyproject.org/community/governance/gtb/) that represents [Working Groups](https://galaxyproject.org/community/wg/) and a [Galaxy Community Board](https://galaxyproject.org/community/governance/gcb/) that represents [Special Interest Groups](https://galaxyproject.org/community/sig/).
 
-### Creating a new SIG
+<p align="center">
+<img src="structure.png" alt="Galaxy Executive Board is in a rectangle over top of two rectangles, the Galaxy Technical Board and Galaxy Community Board, which are themselves over top of Working Groups and Special Interest Groups, respectively. A Project Management Office rectangle spans the image across the bottom" style="width:450px">
+</p>
 
-We encourage everyone with an interest in contributing to Galaxy to join a SIG, WG, or to create their own SIG. The Galaxy Project Management Office (PMO)
-is the first point of contact for anyone wanting to form a new SIG. To illustrate the scope of SIGs, see the community page on the
-Galaxy Community Hub and the following examples:
 
-* Microbial (People interested in Microbial research, tool, workflow and training development)
-* Industry Forum (a forum for people to exchange needs and experiences with Galaxy in Industry settings)
-* Indian Galaxy Community (regional Galaxy community, interested in Galaxy community building)
-* AnVIL (funded project related to Galaxy)
-* BLA (Better License Annotation SIG - does not exist yet, but could bring people together to better annotate Galaxy tools and workflows with appropriate licenses, could develop License recommendations)
+## Who is in the GCB?
+Each Galaxy Special Interest Group (**SIG**) may elect one member to represent their interests on the [GCB](/community/governance/gcb/). In practice, this is often (but not always) the person chairing the **SIG**.
+This is analogous to the [Technical Board](/community/governance/gtb/), consisting of representatives of the [Working Groups (WG)](/community/wg).
 
-## Why make a SIG?
+You can learn more about **SIGs** in our dedicated tutorials below, or explore our [**SIG** directory](/community/sig).
 
-### Prevent work duplication
+<button type="button" onclick="window.open('https://training.galaxyproject.org/training-material/topics/community/tutorials/sig_define/tutorial.html', '_blank')" style="margin-right: 10px">
+  <img src="gtn_hat.png" alt="GTN Hat" width="50">
+  What is a SIG?
+</button>
```

Comment:
```suggestion
<a href="https://training.galaxyproject.org/training-material/topics/community/tutorials/sig_define/tutorial.html" style="margin-right: 10px">
  <img src="gtn_hat.png" alt="GTN Hat" width="50">
  What is a SIG?
</a>
```

would this work? I am uhh very against a button here. It's semanticlaly not a button. (don't want your a11y team to flag it later.)

Reply:
Done!